### ISSUE TITLE
print matrixone and matrixcube version and commit info in log

### ISSUE THREAD

**Issue by lignay** [matrixorigin/matrixone#1286]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

print matrixone and matrixcube version and commit info in log

**Comment by m-schen** [comment]
@broccoliSpicy 
this issue has been solved. Please describe how to use it at comment and Assign this issue to lignay then.

**Comment by broccoliSpicy** [comment]
This issue is solved by commit #1331. Pass the argument "--version" to mo-server executable and you will get the mo-server build information.
